scientific-computing-with-python/learn-data-structures-by-building-the-merge-sort-algorithm/merge-sort-algorithm.py

```diff
@@ -0,0 +1,43 @@
+def mergeSort(array):
+  if len(array) > 1:
+
+      midPoint = len(array)//2
+      leftArray = array[:midPoint]
```

Comment:
Some naming alternative: `left_part`. Not that `left_array` is a bad name, having `x_array` or `array` in almost every line might be harder to more clearly mentally differentiate between how they are used in function. Also technically these are python lists, not arrays.